rllab/dynamics_randomization/tosser.xml

```diff
@@ -0,0 +1,86 @@
+<!--Author: vikash@openai.com-->
```

Comment:
This can be removed, it was only in my sample for testing purposes.

Reply:
If you want to use it in a test, you should be able to find it directly in the mujoco_py package. Or you can use some other mujoco XML file already in our own tree (preferable).

Reply:
You can also just define the XML inline in a string at the top of the file. That's probably the easiest option.